backend/src/test/java/corea/room/service/RoomServiceTest.java

```diff
@@ -44,6 +50,10 @@ class RoomServiceTest {
 
     @Autowired
     private MemberRepository memberRepository;
+    @Autowired
```

Comment:
```suggestion

    @Autowired
```

아래 어노테이션도 상단에 공백 라인 추가해주세요! ☺️